src/pages/SearchPage.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+import ConcertCard from "../components/ConcertCard";
+import Heading from "../components/Header";
+
+SearchPage.propTypes = {
+  pageName: PropTypes.string,
+  concerts: PropTypes.array,
+  onClick: PropTypes.func,
+};
+
+export default function SearchPage({ pageName, onNavigate, concerts }) {
+  return (
+    <>
+      <Heading pageName={pageName} />
+      <ConcertList>
+        {concerts.map((concert) => (
+          <ConcertCard
+            key={concert.id}
+            date={concert.date}
+            artist={concert.artist}
+            location={concert.location}
+            image={concert.image}
+            price={concert.price}
+            onClick={() => onNavigate(concert.id)}
+          />
+        ))}
+      </ConcertList>
+    </>
+  );
+}
+
+const ConcertList = styled.ul`
```

Comment:
Since this is a list, the `ConcertCard` _has to be_ an `li` element! And not a button. Please reconsider it.